defaultModule/requestModules/fcsSetters.md

```diff
@@ -0,0 +1,565 @@
+
+In this scenario, the *attribute* will be null/undefined or simply "enabled". 
+The *value* will be a string containing a boolean "true" or "false". *true* will enable audio description, and *false* will disable it.
+
+**Example**
+
+```fcsSetters.setAudioDescriptionSettings("setEnabled", "true");```
+
+</details>
+
+
+<details id="setClosedCaptions">
+<summary><b>fcsSetters.setClosedCaptions</b></summary>
+
+Configures closed caption settings.
+
+```
+function setClosedCaptions(attribute,value) {
+    let response = // Call your service to update the Closed Captions state for the device
+    let success = // Perform validations to ensure closed caption was updated successfully
+
+    if(success) {
+        return setterSuccess(`Set Closed Captions attribute '${attribute}' to '${value}' successfully!`);
+    } else {
+        return setterFailure(`Unable to set Closed Caption attribute '${attribute}' to '${value}'`);
+    }
+}
+```
+
+There are 2 possible scenarios this function should handle:
+
+**Enabling or Disabling Closed Captions**
+
+In this scenario, the *attribute* will be null/undefined or simply "enable". 
+The *value* will be a string containing a boolean "true" or "false". *true* will enable closed captioning, and *false* will disable it.
+
+**Setting a Closed Captions attribute**
+
+In this scenario, the *attribute* will contain an attribute for which to set a value, and *value* will contain that value.
+
+**Examples**
+
+Enable Closed Captions: ```fcsSetters.setClosedCaptions("true")``` 
+
+Set "FontFamily" to "Arial": ```fcsSetters.setClosedCaptions("FontFamily", "Arial")```
+
+</details>
+
+
+<details id="setDeviceAudio">
+<summary><b>fcsSetters.setDeviceAudio</b></summary>
+
+Configures audio settings for the device.
+
+```
+function setDeviceAudio(value) {
+    let response = // Call your service to set device audio state
+    let success = // Perform validations to ensure device audio was updated successfully
+
+    if (success) {
+        return setterSuccess(`Set Device Audio to '${value}' successfully!`);
+    } else {
+        return setterFailure(`Failed to set Device Audio to '${value}'`);
+    }
+}
+```
+
+Param **value** will represent the desired audio mode.
+
+**Example**
+
+```fcsSetters.setDeviceAudio("Stereo");```
+
+</details>
+
+<details id="setDiscoveryPolicy">
+<summary><b>fcsSetters.setDiscoveryPolicy</b></summary>
+
+Configures the discovery policy for the device.
+
+```
+function setDiscoveryPolicy(value) {
+    let response = // Call your service to set discovery policy state
+    let success = // Perform validations to ensure discovery policy was updated successfully
+
+    if (success) {
+        return setterSuccess(`Set Discovery Policy to '${value}' successfully!`);
+    } else {
+        return setterFailure(`Failed to set Discovery Policy to '${value}'`);
+    }
+}
+```
+
+Param **value** will represent the desired discovery policy.
+
+**Example**
+
+```fcsSetters.setDiscoveryPolicy("Allow");```
```

Comment:
Are the only options "Allow" and (I'm assuming) "Disallow"? If so can we put them here as the two options?